script.js

```diff
@@ -0,0 +1,62 @@
+const addBooks = document.querySelector('.addBook');
+const booksList = document.querySelector('.books-list');
+const remover = document.getElementsByClassName('remover');
+const newB = document.getElementsByClassName('thebook');
+function Book(name, title) {
+  this.name = name;
+  this.title = title;
+  this.id = Math.random();
+}
```

Comment:
- [ ] It seems like you are not using JavaScript classes. Kindly, refactor your code to make sure you are using `class`.
_Consider the following example:_
```
class Books {
  constructor() {
    this.books = [];
  }
    //  Here you will define all the methods. Here's an example.
    
    addbook(title, author) {
      // Code to add the book to `this.books`
    }
 }
```
_Remember, this is just an example_